lib/Bio/EnsEMBL/DataCheck/Checks/CheckChar.pm

```diff
@@ -0,0 +1,85 @@
+=head1 LICENSE
+
+Copyright [2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckChar;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+    NAME        => 'CheckChar',
+    DESCRIPTION => 'Check that imported names/descriptions contains only supported characters',
```

Comment:
This is only relevant for variation databases, so need DB_TYPES => ['variation'] here.
Also, since this datacheck is only concerned with one table, that should be set (i.e. TABLES => ['phenotype']) in order for efficient skipping of tests.